src/math.rs

```diff
@@ -0,0 +1,14 @@
+#[cfg(not(feature = "std"))]
+pub use libm::{roundf, truncf};
+
+#[cfg(feature = "std")]
+#[inline]
+pub fn roundf(x: f32) -> f32 {
+    x.round()
+}
+
+#[cfg(feature = "std")]
+#[inline]
+pub fn truncf(x: f32) -> f32 {
+    x.trunc()
+}
```

Comment:
I went ahead and introduced a math module. The idea is that `round`, `trunc` and co. are sometimes actual instructions, so using the simplistic software implementations in the `libm` crate is not ideal if you have `std` available.